programs/vote/src/vote_state/mod.rs

```diff
@@ -958,10 +959,24 @@ pub fn withdraw<S: std::hash::BuildHasher>(
         .lamports()?
         .checked_sub(lamports)
         .ok_or(InstructionError::InsufficientFunds)?;
+    let received_credits_previous_epoch = match vote_state.epoch_credits.len() {
+        0 => false,
+        _ => {
+            let maybe_last_epoch_credits = vote_state.epoch_credits.last();
```

Comment:
I think we actually want the 2nd to last vec entry here, don't we?
The last epoch is the epoch in progress, so it might only encompass a few slots since the most-recent epoch boundary).